ffi/wgpu.h

```diff
@@ -994,6 +996,8 @@ void wgpu_request_adapter_async(const WGPURequestAdapterOptions *desc,
 
 void wgpu_sampler_destroy(WGPUSamplerId sampler_id);
 
+void wgpu_set_log_callback(int level, WGPULogCallback callback);
```

Comment:
Why not register a function which gets called with each log message in the form of a level/message pair?

Reply:
Yes, this would indeed be better :)  though you also want to set the max log level to avoid the overhead of debug messages being sent over.

Reply:
Yes, true, so the level should be set at reasonable level by default (info or warning).